xchange-okcoin/src/main/java/com/xeiam/xchange/okcoin/dto/trade/OkCoinOrdersResult.java

```diff
@@ -2,11 +2,11 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class OkCoinOrderResult extends OkCoinErrorResult {
+public class OkCoinOrdersResult extends OkCoinErrorResult {
```

Comment:
This is a public class, so it should be kept with old name. Otherwise the new version of the library will be incompatible with the code which relied on the older versions.